editor/inserter/menu.js

```diff
@@ -225,7 +225,15 @@ class InserterMenu extends wp.element.Component {
 	render() {
 		const { position = 'top', instanceId } = this.props;
 		const visibleBlocksByCategory = this.getVisibleBlocksByCategory( wp.blocks.getBlockTypes() );
-		const positionClasses = position.split( ' ' ).map( ( pos ) => `is-${ pos }` );
+		const visualEditorHeight = document.querySelector( '.editor-visual-editor' ).clientHeight;
+		const minimumNeededHeight = 600;
```

Comment:
An issue we might run into here is that the height of the inserter is [dynamic depending on the viewport](https://github.com/WordPress/gutenberg/blob/cd720ecd44c530a9a4a1d513b402cd558fd07b63/editor/inserter/style.scss#L138). Additionally, this could become difficult to maintain if we wanted to adjust the styling, since we'd similarly need to update component's JavaScript as well (and there's an assumption the unknown future maintainer knows to do this).